lib/nets/network.py

```diff
@@ -51,8 +51,10 @@ def __init__(self, batch_size=1):
   def _add_gt_image(self):
     # add back mean
     image = self._image_gt_summaries['image']['placeholder'] + cfg.PIXEL_MEANS
+    #print(image)
     # BGR to RGB (opencv uses BGR)
-    self._gt_image = tf.reverse(image, axis=[-1])
+    self._gt_image = tf.reverse(image, dims=[False, False, False, True])
+    #self._gt_image = tf.reverse(image, axis=[-1])
 
```

Comment:
Backward compatibility to 0.xx is not desired because these ops will be removed soon.